File: nodes-ros2/scripts/node_oop_template/node_oop_template.cpp
#include "rclcpp/rclcpp.hpp"

class MyCustomNode : public rclcpp::Node // MODIFIQUE O NOME
{
public:
  MyCustomNode()
      : Node("node_name") // MODIFIQUE O NOME
  {}

private:
};

int main(int argc, char **argv) {
  rclcpp::init(argc, argv);
  auto node = std::make_shared<MyCustomNode>(); // MODIFIQUE O NOME
  rclcpp::spin(node);
  rclcpp::shutdown();
  return 0;
}

File: nodes-ros2/scripts/node_oop_template/node_oop_template.py
#!/usr/bin/env python3
import rclpy
from rclpy.node import Node


class MyCustomNode(Node): # MODIFIQUE O NOME
    def __init__(self):
        super().__init__("node_name") # MODIFIQUE O NOME


def main(args=None):
    rclpy.init(args=args)
    node = MyCustomNode() # MODIFIQUE O NOME
    rclpy.spin(node)
    rclpy.shutdown()


if __name__ == "__main__":
    main()
